test-cases/features/per-partition-limit.yaml

```diff
@@ -0,0 +1,23 @@
+test_duration: 180
+# using scylla-bench instead cassandra-stress as c-s rapidly retries on failure (e.g. due per-partition limit error)
+# causing overloading cluster due flood of failing requests
+
+stress_cmd_w: "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=1000000 -clustering-row-count=1 -concurrency=50 -connection-count=10 -rows-per-request=1 -consistency-level=quorum -iterations=0 -timeout=300s -duration=10m"
+stress_cmd_r: "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=1000000 -clustering-row-count=1 -concurrency=50 -connection-count=10 -rows-per-request=1 -consistency-level=quorum -iterations=0 -timeout=300s -duration=10m"
```

Comment:
AFAIK scylla-bench, by default, ignores all errors and doesn't limit the number of errors received. We probably want to stop these after receiving an error, right? In that case, maybe we should set `-error-at-row-limit 1` here, or the new `-error-limit 1`.

Reply:
SCT is doing -error-at-row-limit 1000
If nothing else is defined.

But I agreed setting to fail on any error is reasonable here